components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt

```diff
@@ -65,3 +73,22 @@ abstract class DownloadDialogFragment : AppCompatDialogFragment() {
 fun Long.toMegabyteString(): String {
     return String.format("%.2f MB", this / MEGABYTE)
 }
+
+/**
+ * Converts the bytes to kilobytes with two decimal places and returns a formatted string
+ */
+fun Long.toKilobyteString(): String {
+    return String.format("%.2f KB", this / KILOBYTE)
+}
+
+/**
+ * Converts the bytes to megabytes or kilobytes( if size smaller than 0.01 MB)
+ * with two decimal places and returns a formatted string
+ */
+fun Long.toMegaByteOrKilobyteString(): String {
+    return if (this / MEGABYTE < BYTES_TO_MB_LIMIT) {
```

Comment:
Can you please add a bracket around (this / MEGABYTE)? From the operator precedence table it looks like the `<` operator is higher precedence.

Reply:
Multiplicative operators have higher precedence than Comparison operators.

Reply:
Ah my mistake. 😓   Thanks